### ISSUE TITLE
Could not resolve "#app/entry"

### ISSUE THREAD

**Issue by simon-kramer** [nuxt-modules/storybook#670]
As described in the title, i get the error following. I do not get the issue here. The issue only appears after updating to 8.1.11 of storybook. With 8.1.9 the setup runs just fine. Important Note: Its a Nuxt Layer structure with tailwindcss, etc.

```
✘ [ERROR] Could not resolve "#app/entry"

    node_modules/@storybook-vue/nuxt/dist/preview.mjs:7:31:
      7 │   const nuxtApp = () => import('#app/entry').then((m) => m.default);
        ╵                                ~~~~~~~~~~~~

  You can mark the path "#app/entry" as external to exclude it from the bundle, which will remove
  this error and leave the unresolved path in the bundle. You can also add ".catch()" here to handle
  this failure at run-time instead of bundle-time.


 ERROR  error while updating dependencies:                                                                                                                                   10:43:08 AM
Error: Build failed with 1 error:
node_modules/@storybook-vue/nuxt/dist/preview.mjs:7:31: ERROR: Could not resolve "#app/entry"
    at failureErrorWithLog (/Users/layer/ui/node_modules/esbuild/lib/main.js:1472:15)
    at /Users/layer/ui/node_modules/esbuild/lib/main.js:945:25
    at /Users/layer/ui/node_modules/esbuild/lib/main.js:1353:9
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

![Uploaded-using-RayThis-Extension(1)](https://github.com/nuxt-modules/storybook/assets/62208545/0fa06935-d25f-4dce-aacb-4a090c130a4f)
![Uploaded-using-RayThis-Extension(2)](https://github.com/nuxt-modules/storybook/assets/62208545/e27d25a7-3aee-4e51-abb9-3534ac0b29c2)
![Uploaded-using-RayThis-Extension(3)](https://github.com/nuxt-modules/storybook/assets/62208545/bd7bee3c-941f-483e-9e28-8510c2dc7ccd)
![Uploaded-using-RayThis-Extension(4)](https://github.com/nuxt-modules/storybook/assets/62208545/72955967-795c-4e71-8489-82f79c1744ab)

**Comment by NRodriguezcuellar** [comment]
does deleting your node_modules resolve the issue?

**Comment by simon-kramer** [comment]
Sadly it doesn't. I tried every way of debugging it.

**Comment by tobiasdiez** [comment]
This is fixed by https://github.com/nuxt-modules/storybook/pull/619 (not yet released)

**Comment by devbyray** [comment]
> This is fixed by #619 (not yet released)

That is awesome! When will it be released?

**Comment by BloodWorkXGaming** [comment]
We are also running into this issue.
Glad to hear there is a fix! Will it be released soon? If yes, any time estiamte?
If no, is there a workaround for now?

Thank you!

**Comment by tobiasdiez** [comment]
It will be released as soon as Chakir has time for it: https://github.com/nuxt-modules/storybook/issues/667

**Comment by obulat** [comment]
I tried the new release, and I'm now getting new errors:

```
WARN  Sourcemap for "/virtual:/@storybook/builder-vite/setup-addons.js" points to missing source files
WARN  Sourcemap for "/virtual:/@storybook/builder-vite/vite-app.js" points to missing source files

✘ [ERROR] Could not resolve "virtual:nuxt-runtime-config"

    ...@storybook-vue/nuxt/dist/preview.mjs:5:30:
      5 │ import { runtimeConfig } from 'virtual:nuxt-runtime-config';
        ╵                               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
You can mark the path "virtual:nuxt-runtime-config" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.

✘ [ERROR] Could not resolve "#build/css"

    .../@storybook-vue/nuxt/dist/preview.mjs:6:7:
      6 │ import '#build/css';
        ╵        ~~~~~~~~~~~~

You can mark the path "#build/css" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.

✘ [ERROR] Could not resolve "#build/plugins"

    .../@storybook-vue/nuxt/dist/preview.mjs:7:20:
      7 │ import plugins from '#build/plugins';
        ╵                     ~~~~~~~~~~~~~~~~
  You can mark the path "#build/plugins" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.

 ERROR  error while updating dependencies:
Error: Build failed with 3 errors:
.../@storybook-vue/nuxt/dist/preview.mjs:5:30: ERROR: Could not resolve "virtual:nuxt-runtime-config"
.../@storybook-vue/nuxt/dist/preview.mjs:6:7: ERROR: Could not resolve "#build/css"
.../@storybook-vue/nuxt/dist/preview.mjs:7:20: ERROR: Could not resolve "#build/plugins"
    at failureErrorWithLog (/Users/obulat/Documents/openverse/openverse/node_modules/.pnpm/esbuild@0.21.5/node_modules/esbuild/lib/main.js:1472:15)
    at .../.pnpm/esbuild@0.21.5/node_modules/esbuild/lib/main.js:945:25
    at .../.pnpm/esbuild@0.21.5/node_modules/esbuild/lib/main.js:1353:9
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```
_paths redacted_


Here are the storybook-related package versions (from `pnpm list`)

- `@storybook/addon-essentials` 8.2.1
- `@storybook/vue3` 8.2.1 
- `@storybook/addon-interactions` 8.2.1 
- `@storybook/addon-links` 8.2.1
- `@nuxtjs/storybook` 8.1.1 
- `@storybook/blocks` 8.2.1
- `@storybook/builder-vite` 8.2.1 
- `storybook` 8.2.1                     
- `@storybook-vue/nuxt` 0.2.10
- `@storybook/test` 8.2.1

Nuxt version is 3.12.2

Seems that the latest `@nuxtjs/storybook` release is broken: I tried manually replacing ` `8.1.1` version with `8.1.2`, and then I got `Cannot find module '.../@nuxtjs/storybook/dist/module.cjs'` error, and `npm` page lists only `package.json` in the `code` tab for this package: https://www.npmjs.com/package/@nuxtjs/storybook?activeTab=code

**Comment by tobiasdiez** [comment]
@obulat do you get these warnings/errors during the build of this package? In this case, it should be fixed by #695. Or do you get this while *using* the package? In this case, could you please open a new issue? Thanks!
